File: src/main/java/com/itsci/mju/maebanjumpen/skilltype/repository/SkillTypeRepository.java
package com.itsci.mju.maebanjumpen.skilltype.repository;

import com.itsci.mju.maebanjumpen.entity.SkillType;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface SkillTypeRepository extends JpaRepository<SkillType, Integer> {
}

File: src/main/kotlin/com/itsci/mju/maebanjumpen/skilltype/repository/SkillTypeRepository.kt
package com.itsci.mju.maebanjumpen.skilltype.repository

import com.itsci.mju.maebanjumpen.entity.SkillType
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface SkillTypeRepository : JpaRepository<SkillType, Long>{

}
